docs/video-miners/guides/contract-wallet-migration.md

```diff
@@ -1,82 +1,89 @@
-    - `uint256 _maxGas`: This should be the `maxGas` printed by the command line tool
-    - `uint256 _gasPriceBid`: This should be the `gasPriceBid` printed by the command line tool
-    - `uint256 _maxSubmissionCost`: This should be the `maxSubmissionCost`  printed by the command line tool
-- The ETH value to include with the function call should be the `value` printed by the command line tool. **The `value` printed by the command line tool is denominated in Wei so make sure to convert it into the units (i.e. Ether) that is required by the tool you are using to submit the transaction**
-
-### Waiting For Transaction Finalization on L2
-
-After a migration transaction is submitted by the contract wallet and is confirmed on L1, there will be roughly ~10 minutes until the transaction is finalized on L2. During this time you can use a command line tool to monitor for when the transaction is finalized on L2:
-
-```
-# Clone the repository
-git clone https://github.com/livepeer/arbitrum-lpt-bridge
-# Navigate into the repository
-cd arbitrum-lpt-bridge
-# Install dependencies
-yarn
-# Set environment variables
-export MAINNET_URL=<ETHEREUM_MAINNET_RPC_URL>
-export ARB_MAINNET_URL=<ARBITRUM_MAINNET_RPC_URL>
-# Run command
-npx hardhat wait-to-relay-tx-to-l2 <L1_TX_HASH>
-```
-
-`<L1_TX_HASH>` is the hash of the L1 transaction submitted by your contract wallet.
-
-The command will notify you when the transaction is finalized on L2. Then, you should be able to navigate to https://explorer.livepeer.org/ and search for the L2 address that you specified to view the migrated stake owned by that address.
\ No newline at end of file
+---
+title: Contract Wallet Stake Migration Guide
+sidebar_position: 11
+---
+
+This guide provides instructions for Livepeer participants using a contract wallet (i.e., multisig) to stake on L1 (Ethereum Mainnet), to migrate this stake to L2 (Arbitrum Mainnet) as per the [Livepeer Confluence upgrade](https://medium.com/livepeer-blog/the-confluence-upgrade-is-live-3b6b342ea71e). 
+
+This guide is designed to be used in conjunction with the [Livepeer Explorer L2 Contract Wallet Migration Tools](https://explorer.livepeer.org/migrate) to support migrating to Arbitrum: 
+
+- Orchestrators - Migrate stake and fees
+- Broadcasters - Migrate your deposit and reserve
+- Delegators - Migrate undelegated stake 
+
+The migration process will transfer ownership of funds (i.e., stake) owned by your L1 address to the L2 address.
+
+## Create an L2 Address
+
+**Before moving your stake, you MUST confirm that you have a valid L2 address that can own funds migrated from L1. Otherwise, you could lose access to your L1 assets after the migration if you use the L1 address of your contract wallet as the L2 address**
+
+The L2 address can be that of a contract wallet (i.e. [Gnosis Safe](https://gnosis-safe.io/)) deployed on L2, or an EOA, an externally owned account managed by a hardware wallet (e.g., Ledger), or a browser wallet (e.g., Metamask). 

```

Comment:
```
or an EOA, an externally owned account managed by a hardware wallet (e.g., Ledger), or
```

should this be a parenthetical since the clause is describing the EOA?